lib/NodeManager.php

```diff
@@ -57,6 +58,10 @@ public function find($identifier)
             throw new DocumentNotFoundException(sprintf(
                 'Could not find document with ID or path "%s"', $identifier
             ), null, $e);
+        } catch (ItemNotFoundException $e) {
```

Comment:
Test + comment -- why do we recieve different exceptions when a path is not found?